test/validators.js

```diff
@@ -70,7 +70,7 @@ describe('Validators', () => {
         'hans@m端ller.com',
         'test|123@m端ller.com',
         'test123+ext@gmail.com',
-        'some.name.midd.leNa.me+extension@GoogleMail.com',
+        'some.name.midd.leNa.me.and.locality+extension@GoogleMail.com',
```

Comment:
Why it was passing the test before? Is the GoogleMail domain part of gmail?

Reply:
Max length is supposed to be 30 characters without the dots and the email alias (any string after the `+` character) . Even after removing only one dot the old string length was less than 30